src/webgl/p5.RendererGL.Immediate.js

```diff
@@ -221,20 +231,14 @@ p5.RendererGL.prototype._drawFillImmediateMode = function(
   if (this.drawMode === constants.FILL || this.drawMode === constants.TEXTURE) {
     switch (this.immediateMode.shapeMode) {
       case constants.LINE_STRIP:
```

Comment:
@mlarghydracept is the math for calculating the shapes done elsewhere (are they computed to be triangle fans always? or should we mark this part of the code as TODO for future implementation / given issue #2344 ?)

Reply:
I'm not sure about that. i don't think i changed any logic here...

Reply:
Right. Something about not having the individual outcomes explicitly (even though they were all the same) made me wonder if this was done on purpose originally as a stop-gap and maybe doesn't indicate that as clearly, updated in this way. It's functionally the same, though!

Reply:
Hey sorry - my notifications are whacky so I sometimes miss my mentions. No changes are made here. The reason things are happening like they do over in #2344 isn't because of gl modes because gl isn't actually rendering those lines. So, counter-intuitively, the constants.LINE_STRIP has nothing to do with that problem.